src/commands/relay.mjs

```diff
@@ -69,12 +72,27 @@ export class RelayCommand extends BaseCommand {
       throw new MissingArgumentError('Node IDs must be specified')
     }
 
+    const networkJsonString = this.prepareNetworkJsonString(nodeIDs)
+    valuesArg += ` --set config.HEDERA_NETWORK='${networkJsonString}'`
+    return valuesArg
+  }
+
+  // created a json string to represent the map between the node keys and their ids
+  // output example '{"node-1": "0.0.3", "node-2": "0.004"}'
+  prepareNetworkJsonString (nodeIDs = []) {
+    if (!nodeIDs) {
+      throw new MissingArgumentError('Node IDs must be specified')
+    }
+
+    const netWorkIds = {}
+    const accountMap = getNodeAccountMap(nodeIDs)
+
     nodeIDs.forEach(nodeID => {
-      const networkKey = `network-${nodeID.trim()}-0:50211`
-      valuesArg += ` --set config.HEDERA_NETWORK.${networkKey}=0.0.3`
+      const networkKey = `network-${nodeID.trim()}:50211`
```

Comment:
`network-node0:50211` would point to the headless service for the network node.  

I think it would be better to use [accountManager.getNodeServiceMap(namespace)[(https://github.com/hashgraph/solo/blob/37d44f304510d29e8424683814960c707a7fa142/src/core/account_manager.mjs#L256-L256), as this would give you a map that would already give you the accountIds as read from the services [and more see link](https://github.com/hashgraph/solo/blob/37d44f304510d29e8424683814960c707a7fa142/src/core/network_node_services.mjs#L17-L17).   Also, it will give you the GRPC port in case it changes.  It can give you the IP address (haProxyClusterIp) and port (haProxyGrpcPort) of the HAProxy service, which I think is what you should be hitting.  Also, if you want to generate a service name we leverage the render functions in the templates.mjs.

Reply:
updated